### PR TITLE
[Docs] book/zh/src/embed/rust.md

### PR DESCRIPTION
**Pull request by Kodalien**
Signed-off-by: kodalien <kkodalien@gmail.com>

### CODE REVIEW

**Comment by alabulei1** [comment]
LGTM! Thanks for your contribution. @Kodalien 

@hydai Could you help merge this PR? Thanks.

File: docs/book/zh/src/embed/rust.md [hunk 1]
```diff
@@ -1 +1,83 @@
 # Rust SDK
+
+
+你可以通过 WasmEdge Rust SDK 将 WasmEdge 嵌入到 Rust 应用程序中。
+
+WasmEdge Rust SDK 涉及两个 Rust crate [wasmedge-sys](https://crates.io/crates/wasmedge-sys) 和 [wasmedge-rs](https://crates.io/crates/wasmedge- sdk），它们根据不同的原则和不同的目的设计。 wasmedge-sys crate 定义了一组低级 Rust API，其仅仅是对 WasmEdge C API 进行封装，以提供相应的安全接口，而 wasmedge-rs crate 提供更优雅和符合人体工程学的 API，更适合应用程序开发。
```
Comment (hydai) [inline docs/book/zh/src/embed/rust.md:6]: 連結有誤，多了空格：`https://crates.io/crates/wasmedge- sdk`

Comment (hydai) [inline docs/book/zh/src/embed/rust.md:6]: `低級` -> `底層`

File: docs/book/zh/src/embed/rust.md [hunk 2]
```diff
@@ -1 +1,83 @@
...
+
+* [wasmedge-sys](https://crates.io/crates/wasmedge-sys) crate 定义了一组底层 Rust API，其仅仅是对 WasmEdge C API 进行封装，以提供相应的安全接口。 [wasmedge-sys](https://crates.io/crates/wasmedge-sys) 更适合作为基础库，为面向应用的更高层Rust库提供服务。
```
Comment (hydai) [inline docs/book/zh/src/embed/rust.md:8]: Rust 前後要加空白

**Review by hydai — CHANGES_REQUESTED** [review]

**Review by hydai — APPROVED** [review]

**Pull request merged**